docs/addons/prometheus-node-exporter.md

```diff
@@ -0,0 +1,48 @@
+# Prometheus Node Exporter Add-on
+
+This add-on installs [prometheus-node-exporter](https://github.com/prometheus/node_exporter).
+
+prometheus-node-exporter Add-on enables you to measure various machine resources such as memory, disk and CPU utilization. For installations from source you must install and configure it yourself. To enable the node exporter: Enable Prometheus.

```

Comment:
`For installations from source you must install and configure it yourself. To enable the node exporter: Enable Prometheus.` what is about, I am confused?

Reply:
Fixed. Please check in the commit.